src/sentry/static/sentry/app/components/dashboards/widgetQueryForm.tsx

```diff
@@ -29,16 +28,16 @@ const generateOrderOptions = (fields: string[]): SelectValue<string>[] => {
 };
 
 type Props = {
-  api: Client;
-  widgetQuery: WidgetQuery;
   organization: Organization;
   selection: GlobalSelection;
   displayType: Widget['displayType'];
+  queries: WidgetQuery[];
```

Comment:
Should the name of this component change now that it is handling multiple queries?

Reply:
I've renamed it to `WidgetQueriesForm` from `WidgetQueryForm`.